core/sup_tree_core/reduction_log_writer.ex

```diff
@@ -0,0 +1,102 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule AntikytheraCore.ReductionLogWriter do
+  @moduledoc """
+  A `GenServer` that logs processes with increasing reduction.
+  """
+
+  use GenServer
+  alias AntikytheraCore.GearLog.FileHandle
+  alias Antikythera.{Time, ContextId}
+
+  @logger_name     "reduction"
```

Comment:
I think that this variable means not logger but a prefix of a file name.

Reply:
I thought it was rare to change the file name, so I stopped using the module attribute. c27aa9955393893f80ab93e2cc722ff7f674281a